src/EnergyPlus/WindowManager.cc

```diff
@@ -659,42 +659,42 @@ namespace WindowManager {
+                                              state.dataWindowManager->rfPhi[IGlass - 1][ILam - 1],
+                                              state.dataWindowManager->rbPhi[IGlass - 1][ILam - 1],
                                               state.dataWindowManager->lSimpleGlazingSystem,
                                               state.dataWindowManager->SimpleGlazingSHGC,
                                               state.dataWindowManager->SimpleGlazingU);
                         }
                     } else {
                         for (ILam = 1; ILam <= (int)state.dataWindowManager->wle.size(); ++ILam) {
-                            auto lam = state.dataWindowManager->wle(ILam);
-                            state.dataWindowManager->wlt(IGlass, ILam) = lam;
-                            state.dataWindowManager->tPhi(IGlass, ILam) =
+                            auto lam = state.dataWindowManager->wle[ILam - 1];
+                            state.dataWindowManager->wlt[IGlass - 1][ILam - 1] = lam;
+                            state.dataWindowManager->tPhi[IGlass - 1][ILam - 1] =
                                 CurveManager::CurveValue(state, state.dataMaterial->Material(LayPtr).GlassSpecAngTransDataPtr, Phi, lam);
-                            state.dataWindowManager->rfPhi(IGlass, ILam) =
+                            state.dataWindowManager->rfPhi[IGlass - 1][ILam - 1] =
                                 CurveManager::CurveValue(state, state.dataMaterial->Material(LayPtr).GlassSpecAngFRefleDataPtr, Phi, lam);
-                            state.dataWindowManager->rbPhi(IGlass, ILam) =
+                            state.dataWindowManager->rbPhi[IGlass - 1][ILam - 1] =
                                 CurveManager::CurveValue(state, state.dataMaterial->Material(LayPtr).GlassSpecAngBRefleDataPtr, Phi, lam);
                         }
                     }
                     // For use with between-glass shade/blind, save angular properties of isolated glass
                     // for case that all glass layers were input with spectral-average properties
                     //  only used by between-glass shades or blinds
                     if (AllGlassIsSpectralAverage) {
-                        tBareSolPhi(IGlass, IPhi) = state.dataWindowManager->tPhi(IGlass, 1);
-                        tBareVisPhi(IGlass, IPhi) = state.dataWindowManager->tPhi(IGlass, 2);
-                        rfBareSolPhi(IGlass, IPhi) = state.dataWindowManager->rfPhi(IGlass, 1);
-                        rfBareVisPhi(IGlass, IPhi) = state.dataWindowManager->rfPhi(IGlass, 2);
-                        rbBareSolPhi(IGlass, IPhi) = state.dataWindowManager->rbPhi(IGlass, 1);
-                        rbBareVisPhi(IGlass, IPhi) = state.dataWindowManager->rbPhi(IGlass, 2);
+                        tBareSolPhi(IGlass, IPhi) = state.dataWindowManager->tPhi[IGlass - 1][0];
```

Comment:
I would not make this comment here except for the fact that you are already messing around with the indices. I increasingly think that in cases like this you are better off having a single `array<array<struct>>` rather than three separate `array<array<Real64>>`.